### ISSUE TITLE
Newznab Oesn't work on DSM 5.2

### ISSUE THREAD

**Issue by bladehawk** [SynoCommunity/spksrc#1720]
Synology 5.2 does not load mod_rewrite because they, "do NOT provide mod_auth* by default anymore due to DSM patch size limitation". so there seems to be no way to properly get newznab to work.  Can the package be updated to add this or to use some level of workaround?

**Comment by Dr-Bean** [comment]
That info you're looking at is outdated, and has been for quite a while.
Check `etc/httpd/modules`: all the needed modules are there. Even better, `/etc/httpd/conf/httpd.conf-sys` and `/etc/httpd/conf/httpd.conf-user` show that `mod_auth*` and `mod_rewrite` modules are loaded.

As for newnzab, there's a catch. Since DSM5.x, Synology is using php-fpm with Apache. The method that newnab uses to verify `mod_rewrite` is available can't be used together with php-fpm. 
In other words: even though `mod_rewrite` is available and works, newznab won't recognize it as such, and show a warning. The good news is that as far as I know the package still works if you continue with the installation (right?)
As for the bad news: The newznab source is quite old, and isn't updated, so at some point, we're probably better off removing the package from the repository.

**Comment by bladehawk** [comment]
THANK YOU! 

As long as it works,  I was holding off getting a usenet account because I wanted to make sure it worked first. 

Lance Druger
925-330-5852

**Comment by Dr-Bean** [comment]
Err...to be clear, the installation completes, but that's as far as I went here. Before paying money, I would suggest looking into a free trial period from one of the usenet providers, and making sure that the package does what _you_ want it do.

Again, the newznab source code is old. The usefulness of newznab as a whole is dependant on up-to-date regex's to find releases, the rest is just eyecandy and login restrictions. The package does not include a lot of regex's, and the ones that are included are old. So I would suggest to check that out first.

**Comment by Dr-Bean** [comment]
@bladehawk Have you been able to verify the package works for you?

**Comment by bladehawk** [comment]
No,  I got involved with other things and haven't tried it. I'd need a trial news server account and ive been too busy. 

Lance Druger
925-330-5852

**Comment by Dr-Bean** [comment]
I'll close this issue for now then. Reopen if you run into issues with the package, or maybe better, take a look at https://github.com/SynoCommunity/spksrc/issues/249, where the original discussion on newznab takes place.
